### PR TITLE
[#86] Migrate to Bitstyles v4.3.0

### PR DESCRIPTION
**Pull request by hannahbit**
This PR updates to bitstyles version `4.3.0` and adds support for lower versions.

Bitstyles changelog:
https://github.com/bitcrowd/bitstyles/blob/main/CHANGELOG.md#430---2022-05-25

### CODE REVIEW

File: lib/bitstyles_phoenix/bitstyles.ex [hunk 1]
```diff
@@ -14,12 +14,39 @@ defmodule BitstylesPhoenix.Bitstyles do
...
+        "u-bg-" <> variant -> "u-bg--#{variant}"
+        "u-fg-" <> variant -> "u-fg--#{variant}"
+        "u-font-" <> variant -> "u-font--#{variant}"
+        "u-line-height-" <> variant -> "u-line-height--#{variant}"
+        "u-text-" <> variant -> "u-text--#{variant}"
+        "u-border-radius-" <> variant -> "u-round--#{variant}"
+        _ -> class
+      end
+
+    classname(mapping, "4.1.0")
```
Comment (hannahbit) [inline lib/bitstyles_phoenix/bitstyles.ex:48]:
```suggestion
    classname(mapping, "4.0.0")
```

File: test/bitstyles_phoenix/bitstyles_test.exs [hunk 2]
```diff
@@ -4,15 +4,65 @@ defmodule BitstylesPhoenix.BitstylesTest do
...
+      assert classname("u-line-height-min", "4.0.0") == "u-line-height-min"
+      assert classname("u-text-right", "4.0.0") == "u-text-right"
+      assert classname("u-flex-shrink-0", "4.0.0") == "u-flex-shrink-0"
+      assert classname("u-flex-grow-1", "4.0.0") == "u-flex-grow-1"
+      assert classname("u-flex-wrap", "4.0.0") == "u-flex-wrap"
+      assert classname("u-flex-col", "4.0.0") == "u-flex-col"
+      assert classname("u-grid-cols-3", "4.0.0") == "u-grid-cols-3"
+      assert classname("u-col-span-3", "4.0.0") == "u-col-span-3"
+      assert classname("u-col-start-1", "4.0.0") == "u-col-start-1"
+    end
```
Comment (hannahbit) [inline test/bitstyles_phoenix/bitstyles_test.exs:35]: Version `4.0.0` and `4.1.0` should behave the same way. Do we want to have tests for both or rather just for one (`4.0.0`?) ? WDYT?

Comment (andreasknoepfle) [reply test/bitstyles_phoenix/bitstyles_test.exs:35]: I think I'd leave the tests there. But maybe update the `4.0.0` above in the assertions to `4.1.0`

Comment (hannahbit) [reply test/bitstyles_phoenix/bitstyles_test.exs]: 🙈 Changed it! https://github.com/bitcrowd/bitstyles_phoenix/pull/88/commits/3b6d6089c4a6af86435452bee1edfd78aa6cc3b2

**Review by andreasknoepfle — APPROVED** [review]
Awesome work 👏

**Pull request merged**